openshift-kube-apiserver/admission/route/hostassignment/admission.go

```diff
@@ -147,6 +152,9 @@ var _ initializer.WantsExternalKubeClientSet = &hostAssignment{}
 
 func (a *hostAssignment) SetExternalKubeClientSet(clientset kubernetes.Interface) {
 	a.sarClient = clientset.AuthorizationV1().SubjectAccessReviews()
+	a.validationOpts = routecommon.RouteValidationOptions{
```

Comment:
Same question here.

How hard would it be to move this initialisation to a separate setter ?

As we did for https://github.com/openshift/kubernetes/blob/dee15e888b06a2a1c1ac5a4c10f0cba9077e8d57/openshift-kube-apiserver/admission/scheduler/nodeenv/admission.go#L113

Reply:
Addressed